.ci_support/environment.yml

```diff
@@ -22,3 +22,4 @@ dependencies:
 - seekpath =2.1.0
 - spglib =2.0.2
 - structuretoolkit =0.0.6
+- pyxtal =0.5.9
```

Comment:
I am a bit reluctant to make `pyxtal` a required dependency as long as the environment doubles - as discussed in https://github.com/pyiron/structuretoolkit/issues/46 .

Reply:
This is also the reason why the upgrade from `structuretoolkit` is currently delayed https://github.com/pyiron/pyiron_atomistics/pull/1124

Reply:
Sure, this can wait until a new version of pyxtal is released.